tools/appliance/definitions/systemvmtemplate/install_systemvm_packages.sh

```diff
@@ -71,7 +71,7 @@ function install_packages() {
     iptables-persistent \
     libtcnative-1 libssl-dev libapr1-dev \
     python-flask \
-    haproxy \
+    haproxy quagga \
```

Comment:
yes we will need a new system vm template that has quagga preinstalled for ospf functionality to work. Other than that there are no changes and standard template upgrade procedure should apply.